Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 #!groovy
 
 // if removing jdk look at the test made for deploying
-def jdks = ["jdk8", "jdk9", "jdk10", "jdk11"]
```

Comment:
Seems to be a heap of changes in this pr, more than I would have expected to see just for the addition of the MergeFilesTransformer? Should these other changes be in a separate pr?

Reply:
Agreed, this `Jenkinsfile` change should be a separate commit / PR